CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ For simplicity (mostly to make relicensing easier), client code should be also b
 Design standards
 ------------------------------------------------------------------------
 
-We strive to be maximally intuitive and accessible. "That's what they all say", but the currently-popular flat design trend straight-up sacrifices usability for aesthetics, and we try to take the other side of that trade-off.
+We strive to be maximally intuitive and accessible. "That's what they all say," but the currently-popular flat design trend straight-up sacrifices usability for aesthetics, and we try to take the other side of that trade-off.
```

Comment:
Quote formatting is quite the controversial thing in English orthography.

I'm aware the Chicago Manual of Style wants the comma on the inside here, but it's also common style to put the comma wherever it makes most sense. Linking the phrase to the rest of the sentence isn't _part_ of the phrase, so leaving it outside is valid and I prefer it that way.

Reply:
```suggestion
We strive to be maximally intuitive and accessible. Sure, "that's what they all say", but the currently-popular flat design trend straight-up sacrifices usability for aesthetics, and we try to take the other side of that trade-off.
```

It might be clearer like this.